docs/fermi_hubbard/index.md

```diff
@@ -0,0 +1,22 @@
+# Simulation of spin and charge dynamics in the Fermi-Hubbard model
+
+This module contains an implementation of code to simulate the dynamics of the 
+one-dimensional Fermi-Hubbard model on Google's Quantum Computing Service
+
+## Overview
+
+The examples allow to analyze and reproduce the results published 
+in [arXiv:2010.07965](https://arxiv.org/abs/2010.07965) **[quant-ph]**:
+
+  * The [experiment_example.ipynb](fermi_hubbard/experiment_example.ipynb) 
```

Comment:
Sorry, the link is wrong - you need to remove "fermi_hubbard" here and below (experiment_example.ipynb)

Reply:
Fixed, thanks